user/enterprise/operations-manual.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Travis CI Enterprise Operations manual
+layout: en_enterprise
+
+---
+Welcome to the Travis CI Enterprise Operations Manual! This a living document which provides guidelines and suggestions for troubleshooting your Travis CI Enterprise instance. If you have questions about a specific situation, please get in touch with us via [enterprise@travis-ci.com](mailto:enterprise@travis-ci.com).
+
+This document is made of multiple topics. Each topic is a common problem which we've seen ocurring on a regular basis over the time. The section will guide you through it helping you to resolve it. If the problem couldn't be resolved you'll find instructions on how to proceed at the bottom of the document.
+
+Throughout this document we'll be using the following terms to refer to the two components of your Travis CI Enterprise installation:
+
+- `Platform machine`: The virtual machine that runs most of the Travis web components. This is the machine your domain is pointing to.
+- `Worker machine`: The worker machine(s) run your builds.
+
+## Backups
+
+This section explains how you integrate Travis CI Enterprise in your backup strategy. Here, we'll talk about two topics:
+
+- The encryption key
+- The data directories
+
+### Encryption key
+
+Without the encryption key you cannot access the information in your production database. To make sure that you can always recover your database, make a backup of this key:
+
+1. open a ssh connection to the platform machine
+2. run `travis bash`. This will open a bash session with `root` privileges into the Travis container.
+3. Then run `cat /usr/local/travis/etc/travis/config/travis.yml | grep -A1 encryption:`. Create a backup of the value returned by that command by either writing it down on a piece of paper or storing it on a different computer.
+
+> Without this key the information in the database is not recoverable.
```

Comment:
I think we want markup that indicates urgency rather than a quote? I feel like there are examples in the docs, but I can't place them off the top of my head - happy to look if you need a hand

Reply:
Let's check together on that! 

I already searched the docs but couldn't find anything yet.

Reply:
We currently only have "quote" `>` which displays as a Note. we don't have *much* need for warnings on hosted. But I can think of a few uses so we should probably add one. 

The way we do this is by adding a class to the `>` Note

Reply:
Probably best to put the warning *before* the procedure as well